public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    <p><strong>Web development</strong> is a broad term for the work involved in developing a web site for the Internet (World Wide Web) or an intranet (a private network). Web development can range from developing the simplest static single page of plain text to the most complex web-based internet applications, electronic businesses, and social network services. A more comprehensive list of tasks to which <em> web development </em>  commonly refers, may include web engineering, web design, web content development, client liaison, client-side/server-side scripting, web server and network security configuration, and e-commerce development. Among web professionals, "web development" usually refers to the main non-design aspects of building web sites: writing markup and coding. Most recently Web development has come to mean the creation of <strong>content management systems or CMS.</strong> These <em><strong>CMS</strong></em> can be made from scratch, proprietary or open source. In broad terms the <strong>CMS</strong> acts as middleware between the database and the user through the browser. A principle benefit of a <strong>CMS</strong> is that it allows non-technical people to make changes to their web site without having technical knowledge.</p>
+</body>
+</html>
```

Comment:
Manny, I'm impressed! Your code is looking great. 

I know you've been doing your assignments and keeping up with lessons. It will definitely help you stay on track. 

Sometimes you space your elements unevenly, but that's something we all did at the beginning. It's just best to start coding clean now and make a habit of it.

Keep working hard, I'm excited to see you succeed!